packages/xo-web/src/common/action-button.js

```diff
@@ -10,6 +10,8 @@ import UserError from './user-error'
 import { error as _error } from './notification'
 
 export default class ActionButton extends Component {
+  static GO_BACK = 1
```

Comment:
Do you think a boolean can be more suitable ?

Reply:
I do prefer to use numbers since it allows to extends the usage easliy (for example a  RELOAD action) and Javascript can trip users with a falsish value. 
But I'm not settled on this choice and can put a boolean if 's easier to read.

Reply:
I agree a number makes more sense: we're not toggling something on and off, we're saying "do this specific action".